code/modules/vehicles/vehicle.dm

```diff
@@ -374,3 +375,20 @@
 		new /obj/effect/decal/cleanable/blood/oil(src.loc)
 	spawn(1) healthcheck()
 	return 1
+
+/obj/vehicle/can_fall(turf/below, turf/simulated/open/dest = src.loc)
```

Comment:
how does this handle something like the cargo tug falling while still attached to the trolley?

Reply:
The rest of the trolleys follow when you move down.